### PR TITLE
chore: Update E2E [fixes FLU-297 and FLU-252 and FLU-314 and FLU-298]

### PR DESCRIPTION
**Pull request by nahsi**

### CODE REVIEW

File: .github/workflows/e2e.yml [hunk 1]
```diff
@@ -3,192 +3,296 @@ name: "e2e"
...
-      rust-peer-image: "${{ needs.rust-peer.outputs.rust-peer-image }}"
+      rust-peer-image: "${{ needs.rust-peer-snapshot.outputs.rust-peer-image }}"
+
+  # spell-aqua-tests:
+  #   name: "spell"
+  #   needs:
+  #     - rust-peer-snapshot
+  #   uses: fluencelabs/spell/.github/workflows/tests.yml@main
+  #   with:
+  #     rust-peer-image: "${{ needs.rust-peer-snapshot.outputs.rust-peer-image }}"
```
Comment (ValeryAntopol) [inline .github/workflows/e2e.yml:268]: lets remove or enable it?

Comment (nahsi) [reply .github/workflows/e2e.yml:268]: I will be fixed soon and I will uncomment it

Comment (ValeryAntopol) [reply .github/workflows/e2e.yml:268]: Then comment the commented code with this message 👍

**Review by ValeryAntopol — APPROVED** [review]
utACK

**Pull request merged**